### PR TITLE
Update openstack developer page

### LINKED ISSUE #3200: Updates to Openstack developer page
https://docs.google.com/document/d/1QIXOAlCDV2cyQqU5uEW2kpITobf7OkFHr-tsEvjd_1Q/edit

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Update page according to the [copy doc](https://docs.google.com/document/d/1QIXOAlCDV2cyQqU5uEW2kpITobf7OkFHr-tsEvjd_1Q/edit#)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/openstack/developer>
- Check that changes have been made as per the [copy doc](https://docs.google.com/document/d/1QIXOAlCDV2cyQqU5uEW2kpITobf7OkFHr-tsEvjd_1Q/edit#) - please resolve comments as you verify they are correct

## Issue / Card
Fixes #3200

### CODE REVIEW

**Comment by frankiegranato** [comment]
We are waiting for input from James Page. This page can't be released until we have that.

File: templates/openstack/developer.html [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
+{% block meta_description %}Step-by-step instructions to deploy a full OpenStack cluster on your workstation using LXD machine containers for service isolation.{% endblock %}
 
 {% block content %}
 <section class="p-strip--light is-deep is-bordered">
   <div class="row">
     <div class="col-6">
       <div>
-        <h1>Developer OpenStack</h1>
-        <p>Install OpenStack on a single machine with Conjure-up. It deploys a real-world OpenStack on a single machine using LXD, that perfectly models a production cluster on real machines.</p>
+        <h1>Single-node Developer OpenStack cluster</h1>
```
Comment (pmahnke) [inline templates/openstack/developer.html]: I would put the h1 in its own row.

File: templates/openstack/developer.html [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
+        <p>Install a full OpenStack cluster on a single machine.This guide uses <a class="p-link--external" href="https://linuxcontainers.org/">LXD</a>, the Linux Container Daemon, to create multiple machine containers for the various OpenStack services, mimicking a real production distributed Openstack cluster.</p>
+        <p>Follow these step-by-step instructions.</p>
+        <p><strong>conjure-up</strong> requires a minimum version of LXD of <strong>3.0.0</strong>. Additionally, LXD should be configured prior to running.</p>
       </div>
     </div>
     <div class="col-6">
```
Comment (pmahnke) [inline templates/openstack/developer.html]: The 'u-align--center u-vertically-center u-hide--small' needs to be on the div above, not the image

Comment (pmahnke) [reply templates/openstack/developer.html]: the /openstack/install page likely needs to match this

File: templates/openstack/developer.html [hunk 3]
```diff
@@ -87,16 +82,21 @@ <h3 class="p-list-step__title"><span class="p-list-step__bullet">2</span>Install
...
       <li class="p-list-step__item col-12">
-        <h3 class="p-list-step__title"><span class="p-list-step__bullet">3</span>Select the ‘OpenStack with NovaLXD’ spell</h3>
+        <h3 class="p-list-step__title"><span class="p-list-step__bullet">3</span>Deploy OpenStack</h3>
         <div class="p-list-step__content">
+          <p>Now you are ready to start the OpenStack deployment process on your workstation.</p>
+          <div class="p-code-snippet">
+            <input aria-label="code snippet" class="p-code-snippet__input" value="conjure-up" readonly="readonly">
+            <button class="p-code-snippet__action">Copy to clipboard</button>
+          </div>
+          <p>You will see a menu of ‘spells’ which are descriptions of distributed software systems that conjure-up can deploy for you. Choose ‘OpenStack with LXD’.</p>
```
Comment (pmahnke) [inline templates/openstack/developer.html]: perhaps just straight quotes?

**Comment by steverydz** [comment]
@pmahnke Changes done - also updated the install page in that PR (#3217)

**Pull request merged**